README.md

```diff
@@ -161,27 +99,28 @@ An initial outline that will be updated as needed:
 
 ---
 
+## Origin and history
 
-Version 1 of this doc [can be found here](https://docs.google.com/document/d/1omAxwvCSRlo_u5UL3ThTXFhNccDuN7GJOi4RlZfk--w/edit#heading=h.75nvyav1r98b) for reference.
+The process to create the guide started in the summer of 2020. The first drafts of this document first existed as Google Docs, here are [Version 1](https://docs.google.com/document/d/1omAxwvCSRlo_u5UL3ThTXFhNccDuN7GJOi4RlZfk--w/edit#heading=h.75nvyav1r98b) and [Version 2](https://docs.google.com/document/d/1YiYeRIybGmxmErCOI4Jc8Qajz3JGM1JYVfUtpzyCzSk/edit?usp=sharing) for reference.
 
 
 ---
 
 
-## Run Locally
+## How to build and run the site locally
```

Comment:
Just a note that altering this will require a link change in #113 specifically [here](https://deploy-preview-113--sad-borg-390916.netlify.app/guide/contribute/illustration-guidelines/#file-size-optimization)

Reply:
Updated that link now that #113 is merged.